onboarding/fibonacci.py

```diff
@@ -1,18 +1,30 @@
 def fibonacci_term(n):
-    '''
-    Finds nth term of the Fibonacci sequence from:
-        Term | value
-        ------------
-          1  |   1
-          2  |   1
-          3  |   2
-          4  |   3
-          5  |   5
-         ... |  ...
+  '''
+  Finds nth term of the Fibonacci sequence from:
+       Term | value
+       ------------
+         1  |   1
+         2  |   1
+         3  |   2
+         4  |   3
+         5  |   5
+        ... |  ...
 
-    @params n: integer
-    @returns integer, nth term of the Fibonacci sequence
-    '''
+  @params n: integer
+  @returns integer, nth term of the Fibonacci sequence
+  '''
 
-    # insert code here
-    pass
+  # insert code here
+  if n == 0:
```

Comment:
`n` cannot be zero so the test case should raise a `TypeError`. Also need to check if `n` is an integer and if `n` is an integer, is it <= 0 in which case we want to raise a `TypeError`.